Source/Core/Check.js

```diff
@@ -0,0 +1,144 @@
+/*global define*/
+define([
+    './defaultValue',
+    './defined',
+    './DeveloperError',
+    './isArray'
+    ], function(
+        defaultValue,
+        defined,
+        DeveloperError,
+        isArray) {
+    'use strict';
+
+    /**
+     * Contains functions for checking that supplied arguments are of a specified type
+     * or meet specified conditions
+     * @private
+     *
+     * @exports Check
```

Comment:
I would remove `@exports` since this is private.  This isn't how we typically document a class anyway.  Have you seen the Documentation Guide: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/DocumentationGuide/README.md#documentation-guide